types/react-redux/index.d.ts

```diff
@@ -25,7 +25,7 @@ interface ComponentDecorator<TOwnProps, TMergedProps> {
  * Can't use the above decorator because it would default the type to {}
  */
 export interface InferableComponentDecorator<TOwnProps> {
```

Comment:
I think `TOwnProps` is now a misleading name, since that role is now played by `T`... right?